lib/rollbar.rb

```diff
@@ -21,6 +21,56 @@
 require 'rollbar/delay/thread'
 require 'rollbar/truncation'
 
+a = "foo bar"
+
+a.upcase
+
+valid_lambda = lambda { 'foo' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.